package.json

```diff
@@ -29,11 +29,24 @@
     "react-qr-reader": "^2.2.1",
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.2",
+    "react-scripts": "^4.0.3",
     "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
-    "serve": "^11.3.2"
+    "serve": "^11.3.2",
+    "workbox-background-sync": "^5.1.3",
```

Comment:
Why so many? It seems like you use 3 in the service worker

Reply:
I think it's better to minimize. npm i can get very long, and it also makes the `package.json` easier to maintain. I think CRA is a great starting point, but as an application evolves and grows, it's good to change things to adapt it.

Reply:
Yep, I was about to comment the same about this dependencies. I would also add only those which are necessary.

Reply:
Done.